lib/exercism/language.rb

```diff
@@ -1,16 +1,11 @@
-# rubocop:disable Lint/RescueException
-# Allow all exceptions to be reported to Bugsnag
-
 class Language
-  def self.of(id)
-    by_track_id[id.to_s.downcase]
-  rescue Exception => e
-    Bugsnag.notify(e, track: id)
```

Comment:
What was the usecase being handled by this rescue block? Just wondering why its no longer a concern to track exceptions for this.